ui/analyse/src/study/relay/relayView.ts

```diff
@@ -78,20 +78,20 @@ export function renderStreamerMenu(relay: RelayCtrl) {
   );
 }
 
-function renderBoardView(ctx: ViewContext) {
+function renderBoardView(ctx: RelayViewContext) {
   const { ctrl, deps, study, gaugeOn, relay } = ctx;
   // no worries about FOUC when overriding grid constraints here, it's all snab
   const cols = Number(window.getComputedStyle(document.body).getPropertyValue('--cols'));
   if (cols >= 2)
```

Comment:
Since the beginning of ui/analyse, and throughout the recent broadcast rewrite, I managed to keep the exact same DOM for all devices. It has not been easy but I thought it was worth it.

Reply:
yeah i think we can do it with grid. But it might not be a good idea to bring that lobby alignment mess to analyse.